### PR TITLE
benefits updated

### PR DESCRIPTION
**Pull request by ronitblenz**
As prescribed, I have updated the benefits section.

### CODE REVIEW

File: project-progression-policy.md [hunk 1]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
 4. Project Support: The Consortium can provide limited funds to support project communication and infrastructure costs for CI/CD.
 5. Outreach: Part of the Consortium's mission is to promote the use of confidential computing with various communities and hence its recognized projects, to deliver that outcome.
 6. Internships: The [TAC has committed](https://lists.confidentialcomputing.io/g/main/files/TAC/Meetings/2021/09-Sept/TAC%20Minutes%202021-09-09.pdf) to fund one intern per CCC project that commits to the list of "Mentor Responsibilities" on the [Outreachy mentor page](https://www.outreachy.org/mentor/).
+7. Stipend: Upto $7500 in budget for hardware and software per year.
```
Comment (dthaler) [inline project-progression-policy.md:23]:
```suggestion
7. Stipend: Up to $7500 in budget for hardware and software per year.
```

File: project-progression-policy.md [hunk 2]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
...
+8. Funding: Provided with a funding for one Outreachy Intern.
```
Comment (dthaler) [inline project-progression-policy.md:24]:
```suggestion
8. Intern: Up to one [Outreachy](https://www.outreachy.org) Intern per year.
```

File: project-progression-policy.md [hunk 3]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
...
+9. Mentorship: TAC(Technical Advisory Committee) mentor will be assigned to the project to guide through the edgecases.
```
Comment (dthaler) [inline project-progression-policy.md:25]:
```suggestion
9. Mentorship: The Technical Advisory Council will assign a [mentor](project-mentors.md) to the project.
```

File: project-progression-policy.md [hunk 4]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
...
+10. Collaboration: Interns are provided with [Collaboration tools](mailto:operations@confidentialcomputing.io).
+Meetings will also take place through "Zoom" for further discussion within mentees and mentors.
+11. Domain: Registration and renewals are granted.
```
Comment (dthaler) [inline project-progression-policy.md:28]:
```suggestion
11. Domain name: A domain name registration and renewals can be granted.
```

**Comment by ronitblenz** [comment]
@dthaler Thanks for rectifying me. I have fixed the problems.

**Review by dthaler — APPROVED** [review]

File: project-progression-policy.md [hunk 5]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
 4. Project Support: The Consortium can provide limited funds to support project communication and infrastructure costs for CI/CD.
 5. Outreach: Part of the Consortium's mission is to promote the use of confidential computing with various communities and hence its recognized projects, to deliver that outcome.
 6. Internships: The [TAC has committed](https://lists.confidentialcomputing.io/g/main/files/TAC/Meetings/2021/09-Sept/TAC%20Minutes%202021-09-09.pdf) to fund one intern per CCC project that commits to the list of "Mentor Responsibilities" on the [Outreachy mentor page](https://www.outreachy.org/mentor/).
+7. Test infrastructure: Up to $7500 in budget for hardware and software per year.
+8. Intern: Up to one [Outreachy](https://www.outreachy.org) Intern per year.
```
Comment (dcmiddle) [inline project-progression-policy.md:24]: Isn't this redundant with # 6 in the list?

File: project-progression-policy.md [hunk 6]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
...
+9. Mentorship: The Technical Advisory Council will assign a [mentor](project-mentors.md) to the project.
+10. Collaboration: Interns are provided with [Collaboration tools](mailto:operations@confidentialcomputing.io).
+Meetings will also take place through "Zoom" for further discussion within mentees and mentors.
+11. Domain name: A domain name registration and renewals can be granted.
+12. Mailing lists: For further networking.
+13. YouTube Playlist: For reference and learning.
+14. Security Scanning: Optional.
+15. LFX tools: The most important thing are the [tools](https://lfx.linuxfoundation.org/).
```
Comment (dcmiddle) [inline project-progression-policy.md:32]: I don't understand the "most important" comment. Maybe instead something like... 
```suggestion
15. LFX tools: Access to Linux Foundation tools to improve many aspects of a project from security to community health [tools](https://lfx.linuxfoundation.org/).
```

**Review by dcmiddle — CHANGES_REQUESTED** [review]

File: project-progression-policy.md (hunk 5 above)
Comment (ronitblenz) [reply project-progression-policy.md]: I have removed the redundant listings.

File: project-progression-policy.md (hunk 6 above)
Comment (ronitblenz) [reply project-progression-policy.md]: Commited changes

File: project-progression-policy.md [hunk 7]
```diff
@@ -20,6 +20,14 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
 4. Project Support: The Consortium can provide limited funds to support project communication and infrastructure costs for CI/CD.
 5. Outreach: Part of the Consortium's mission is to promote the use of confidential computing with various communities and hence its recognized projects, to deliver that outcome.
 6. Internships: The [TAC has committed](https://lists.confidentialcomputing.io/g/main/files/TAC/Meetings/2021/09-Sept/TAC%20Minutes%202021-09-09.pdf) to fund one intern per CCC project that commits to the list of "Mentor Responsibilities" on the [Outreachy mentor page](https://www.outreachy.org/mentor/).
+7. Test infrastructure: Up to $7500 in budget for hardware and software per year.
+8. Collaboration: Interns are provided with [Collaboration tools](mailto:operations@confidentialcomputing.io).
```
Comment (dcmiddle) [inline project-progression-policy.md:24]: In the slide referenced from the issue there are 4 tools listed. 3 are already in this document. Just need to list zoom.
```suggestion
8. Video conferencing support: A zoom account for community meetings.
```

File: project-progression-policy.md [hunk 8]
```diff
@@ -20,6 +20,14 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
...
+Meetings will also take place through "Zoom" for further discussion within mentees and mentors.
+9. Domain name: A domain name registration and renewals can be granted.
+10. Mailing lists: For further networking.
+11. YouTube Playlist: For reference and learning.
+12. Security Scanning: Optional.
+13. LFX tools: Access to Linux Foundation tools to improve many aspects of a project from security to community health [tools](https://lfx.linuxfoundation.org/).
```
Comment (dcmiddle) [inline project-progression-policy.md:30]:
```suggestion
13. LFX tools: Access to Linux Foundation tools to improve many aspects of a project from security to community health [tools](https://lfx.linuxfoundation.org/).

Approved projects can [contact](mailto:operations@confidentialcomputing.io) for access to these benefits.
```

**Review by dcmiddle — CHANGES_REQUESTED** [review]

**Comment by ronitblenz** [comment]
@dcmiddle I have made further changes.

File: project-progression-policy.md [hunk 9]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
 4. Project Support: The Consortium can provide limited funds to support project communication and infrastructure costs for CI/CD.
 5. Outreach: Part of the Consortium's mission is to promote the use of confidential computing with various communities and hence its recognized projects, to deliver that outcome.
 6. Internships: The [TAC has committed](https://lists.confidentialcomputing.io/g/main/files/TAC/Meetings/2021/09-Sept/TAC%20Minutes%202021-09-09.pdf) to fund one intern per CCC project that commits to the list of "Mentor Responsibilities" on the [Outreachy mentor page](https://www.outreachy.org/mentor/).
+7. Test infrastructure: Up to $7500 in budget for hardware and software per year.
+8. Video conferencing support: A zoom account for community meetings.
+Meetings will also take place through "Zoom" for further discussion within mentees and mentors.
```
Comment (dcmiddle) [inline project-progression-policy.md:25]:
```suggestion
```

File: project-progression-policy.md [hunk 10]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
...
+9. Domain name: A domain name registration and renewals can be granted.
+10. Mailing lists: For further networking.
```
Comment (dcmiddle) [inline project-progression-policy.md:27]:
```suggestion
10. Mailing lists: For asynchronous community discussions.
```

File: project-progression-policy.md [hunk 11]
```diff
@@ -20,6 +20,16 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
...
+11. YouTube Playlist: For reference and learning.
```
Comment (dcmiddle) [inline project-progression-policy.md:28]:
```suggestion
11. YouTube Playlist: For publishing community meetings, demos, and other project content.
```

**Review by dcmiddle — CHANGES_REQUESTED** [review]
Sorry for the incremental feedback. It would have been better had I given this all at once. I think this is the last revision set though. :)

**Comment by ronitblenz** [comment]
@dcmiddle It's completely alright, I made mistakes in the contribution. Thank you for correcting time and again!

**Comment by ronitblenz** [comment]
Kindly merge the pr if it's alright, else suggest me changes. So that I can make a valid contribution.

**Comment by dcmiddle** [comment]
Thanks for the contribution!

**Comment by ronitblenz** [comment]
My pleasure. Will do more

**Pull request merged**